command/exec.go

```diff
@@ -56,6 +56,10 @@ type subprocessExec struct {
 	// allows following commands to execute even if this shell command fails.
 	ContinueOnError bool `mapstructure:"continue_on_err"`
 
+	// NoStripBlankArgs will allow blank arguments in commands if set to true
```

Comment:
I only understand what blank means because of our slack discussion. I think we should be more precise, maybe `NoStripEmptyStringArgs`?

Reply:
renamed to NoStripEmptyArgs